src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,141 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import { lucidClassNames } from '../../util/style-helpers';
+import { buildHybridComponent } from '../../util/state-management';
+import {
+	ExpanderPanelDumb as ExpanderPanel,
+	IExpanderPanelHeaderProps,
+} from '../ExpanderPanel/ExpanderPanel';
+import {
+	findTypes,
+	omitProps,
+	StandardProps,
+	Overwrite,
+} from '../../util/component-types';
+import * as reducers from '../Accordion/Accordion.reducers';
+
+const cx = lucidClassNames.bind('&-Accordion');
+
+const { any, func, object, number, string } = PropTypes;
+
+interface IAccordionPropsRaw extends StandardProps {
+	/**
+	 * Indicates which item is expanded
+	 * */
+	selectedIndex: number;
+	/**
+	 * Called when the user clicks on the component's header of an item.
+	 * */
+
+	onSelect: (
+		selectedIndex: number | null,
+		{ event, props }: { event: React.MouseEvent; props: IAccordionProps }
+	) => void;
+	/**
+	 * Prop alternative to Header child component passed through to the
+	 * underlying ExpanderPanel
+	 */
+	Header: IExpanderPanelHeaderProps;
+}
+
+type IAccordionProps = Overwrite<
+	React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>,
+	IAccordionPropsRaw
+>;
+
+export interface IAccordionState {
+	selectedIndex: number | null;
+}
+
+const defaultProps = {
+	onSelect: _.noop,
+};
+
+class Accordion extends React.Component<IAccordionProps, IAccordionState> {
+	static displayName = 'Accordion';
+	static propTypes = {
+		className: string`
+			Appended to the component-specific class names set on the root element.
+		`,
+		selectedIndex: number`
+			Indicates which item is expanded
+		`,
+		onSelect: func`
+			Called when the user clicks on the component's header of an item.
+		`,
+		style: object`
+			Passed through to the root element.
+		`,
+		Header: any`
```

Comment:
After reviewing the component  I think that is actually wrong at the `Accordion` level. The header is actually at the child level if you look through the examples. I think we can just remove this one from here as well from the type signature.

Reply:
That is a good point. It doesn't seem to even work if you pass a header as a prop at the `Accordion` level. However, removing the prop causes a common test to fail. Seems like this test checks for child components defined as static properties, and makes sure a prop exists for them:
https://github.com/appnexus/lucid/blob/ebe298ecf66b87457fa2665a0de2dd3cbcaea39f/src/util/generic-tests.jsx#L136

Reply:
Looks like you can pass an `exemptChildComponents` prop to the common tests to bypass that.